Trackers/centroid_tracker/centroid.py

```diff
@@ -87,23 +87,12 @@ def update(self, detections):
                 objectId.append(track.id)
                 objectCentroid.append(track.centroid) 
             D = dist.cdist(np.array(objectCentroid), inputCentroids)
-            rows = D.min(axis=1).argsort()
-            cols = D.argmin(axis=1)[rows]
-            usedRows = set()
-            usedCols = set()
-            # loop over the combination of the (row, column) index tuples
-            for (row, col) in zip(rows, cols):
-                # if we have already examined either the row or column value before, ignore it
-                if row in usedRows or col in usedCols:
-                    continue
-                self.tracks[row].bbox = detections[col]
-                self.tracks[row].centroid = inputCentroids[col]
-                self.tracks[row].miss = 0
-                # indicate that we have examined each of the row and column indexes, respectively
-                usedRows.add(row)
-                usedCols.add(col)
-            unusedRows = set(range(0, D.shape[0])).difference(usedRows)
-            unusedCols = set(range(0, D.shape[1])).difference(usedCols)
+            unusedRows,unusedCols,matched= self.find_match(objectCentroid,inputCentroids)
```

Comment:
Instead of using `unusedRows`, `unusedCols` it better to used `matches`, `unmatched_tracks`, `unmatched_detections` in the same order to have a uniform naming standard.